packages/headless/src/controllers/smart-snippet-questions-list/headless-smart-snippet-questions-list.ts

```diff
@@ -72,30 +82,57 @@ export function buildSmartSnippetQuestionsList(
   }
 
   const controller = buildController(engine);
+  const getState = () => engine.state;
+
+  const getIsExpanded = (
+    questionAndAnswer: QuestionAnswer,
+    relatedQuestions: QuestionAnsweringRelatedQuestionState[]
+  ) => {
+    const idx = findRelatedQuestionIdx(
+      relatedQuestions,
+      questionAndAnswer.documentId
+    );
+    if (idx === -1) {
+      return false;
+    }
+    return relatedQuestions[idx].expanded;
+  };
 
   return {
     ...controller,
 
     get state() {
+      const state = getState();
+
       return {
-        // TODO
-      } as SmartSnippetQuestionsListState;
+        relatedQuestions: state.search.response.questionAnswer.relatedQuestions.map(
+          (relatedQuestion) => ({
+            question: relatedQuestion.answerSnippet,
+            answer: relatedQuestion.question,
```

Comment:
Shouldn't these be swapped? 
`question: relatedQuestion.question,
answer: relatedQuestion.answerSnippet,`

Reply:
Ahaha, yep ! 🤦